src/pages/Travel/DomainSelectorPage.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useCallback, useMemo, useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import TravelDomainListItem, {DomainItem} from '@components/SelectionList/TravelDomainListItem';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {userHasLoginInDomain} from '@libs/LoginUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import {getAdminsPrivateEmailDomains, getMostFrequentEmailDomain} from '@libs/PolicyUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+function DomainSelectorPage() {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const [activePolicyID] = useOnyx(ONYXKEYS.NVP_ACTIVE_POLICY_ID);
+    const [selectedDomain, setSelectedDomain] = useState('');
```

Comment:
NAB, but I would probably start this value off at `null` just to be extra clear that nothing has been selected yet.

Reply:
fixed.